File: api/pastassignments.js
const { default: axios } = require("axios");

module.exports = async (req, res) => {
    res.setHeader('Access-Control-Allow-Origin', '*');
    const { username, password, quarter } = req.query;

    const { data } = await axios.get(`https://gradualgrades.herokuapp.com/students/currentclasses?username=${username}&password=${password}&quarter=${quarter}`);

    return res.send(data)

    return res.status(500).send(error.response.data)
}

File: api/pastassignments.go
package handler

import (
	"fmt"
	"net/http"

	"github.com/go-resty/resty/v2"
)

func PastAssignmentsHandler(w http.ResponseWriter, r *http.Request) {
	var restyClient = resty.New()

	queryParams := r.URL.Query()

	username := queryParams.Get("username")
	password := queryParams.Get("password")
	quarter := queryParams.Get("quarter")

	apiURL := fmt.Sprintf("https://gradualgrades.herokuapp.com/students/pastassignments?username=%v&password=%v&quarter=%v", username, password, quarter)

	apiResponse, _ := restyClient.R().
			EnableTrace().
			Get(apiURL)

	w.Header().Add("Content-Type", "application/json") 
	fmt.Fprintf(w, apiResponse.String())
}